src/main/java/core/basesyntax/Figures.java

```diff
@@ -1,14 +1,12 @@
 package core.basesyntax;
 
-abstract class Figures implements Figure {
+public abstract class Figures implements AreaCalculation, Drawable {
```

Comment:
```suggestion
public abstract class Figure implements AreaCalculation, Drawable {
```
the class name should be singular